net.sf.nootka.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "net.sf.nootka",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "nootka",
+  "branch": "1.4.5",
```

Comment:
on flathub everything has to be branch: stable - just remove this bit.

Reply:
Ok, done